File: src/CHTL/Lexer/TokenType.java
package CHTL.Lexer;

public enum TokenType {
    // Literals
    IDENTIFIER,
    STRING,
    NUMBER,

    // Symbols
    LBRACE, RBRACE,
    COLON,
    EQUALS,
    SEMICOLON,

    // Keywords from the spec
    TEXT,
    STYLE,
    SCRIPT,

    EOF
}

File: CHTL/CHTLLexer/token_type.py
from enum import Enum, auto

class TokenType(Enum):
    # Core syntax tokens
    LBRACE = auto()         # {
    RBRACE = auto()         # }
    COLON = auto()          # :
    SEMICOLON = auto()      # ;
    EQUALS = auto()         # =

    # Literals
    IDENTIFIER = auto()
    STRING = auto()

    # Keywords
    TEXT = auto()

    # Special
    EOF = auto()
